libs/designer/src/lib/core/utils/parameters/dynamicdata.ts

```diff
@@ -87,7 +87,7 @@ export async function getDynamicValues(
     return getListDynamicValues(
       connectionReference?.connection.id,
       operationInfo.connectorId,
-      operationInfo.operationId,
+      dynamicState.operationId,
```

Comment:
full dynamic state is already passed into getListDynamicValues, look at the last argument, so you should not be sending this operationId, this breaks the contract and the services implementing it. Maybe your service is not using the correct operationId to make the actual dynamic calls, please check la standard dynamic service.

Reply:
I can revert this as it's not required to make the cache-key unique